docs/ug/cli.md

```diff
@@ -87,7 +87,7 @@ Binary file formats, such as `jpg`, `png`,`exe`,`zip`, `rar`, `docx`, and `pptx`
 <box type="info" seamless>
 
 * This flag only works on **git `2.23`** or later.
-* If an earlier version of **git** is used, RepoSense can still run but this flag will be ignored.
+* If an earlier version of **git** is used, RepoSense can still run but only if this flag is not used.
 </box>
```

Comment:
I believe we can remove this section since we are assuming version 2.23.

Reply:
Thanks for pointing this out, will make the changes accordingly.